assignment/tool/src/webapp/vm/assignment/chef_assignments_student_view_submission.vm

```diff
@@ -555,7 +555,7 @@ $(document).ready(function(){
 							<span style="font-size: smaller; color: blue;">	
 								#set($gusers = $!service.getSortedGroupUsers($group))
 								#foreach ($user in $gusers)
-									$user.getDisplayName()
+									$user.getDisplayName() ($user.getDisplayId())
```

Comment:
I think this should be something like... $validator.escapeHtml($user.DisplayName) ($validator.escapeHtml($user.DisplayId), right?

Reply:
Modified. Thanks @juanjmerono!